### PR TITLE
[Maintenance] Minor cs fixes

### PR DESCRIPTION
**Pull request by lchrusciel**

### CODE REVIEW

File: src/EventListener/CartBlamerListener.php [hunk 1]
```diff
@@ -1,14 +1,5 @@
...
 
-/*
- * This file is part of the Sylius package.
- *
- * (c) Paweł Jędrzejewski
- *
- * For the full copyright and license information, please view the LICENSE
- * file that was distributed with this source code.
- */
-
```
Comment (Tomanhez) [inline src/EventListener/CartBlamerListener.php]: Why you removed it?

File: src/EventListener/CartBlamerListener.php [hunk 2]
```diff
@@ -1,14 +1,5 @@
 <?php
```
Comment (Tomanhez) [inline src/EventListener/CartBlamerListener.php]: redundant space

**Comment by mamazu** [comment]
@lchrusciel You know that we have labels? :D

**Comment by GSadee** [comment]
Thanks, Łukasz! :1st_place_medal:

**Pull request merged**